.github/STALE-BOT.md

```diff
@@ -3,7 +3,7 @@
 - Thanks for your contribution!
 - To remove the stale label, just leave a new comment.
 - _How to find the right people to ping?_ &rarr; [`git blame`](https://git-scm.com/docs/git-blame) to the rescue! (or GitHub's history and blame buttons.)
-- You can always ask for help on [our Discourse Forum](https://discourse.nixos.org/) or on the [#nixos IRC channel](https://webchat.freenode.net/#nixos).
+- You can always ask for help on [our Discourse Forum](https://discourse.nixos.org/), [our Matrix space](https://matrix.to/#/!MKvhXlSTLGJUXpYuWF:nixos.org), or on the [#nixos IRC channel](https://web.libera.chat/#nixos).
```

Comment:
```suggestion
- You can always ask for help on [our Discourse Forum](https://discourse.nixos.org/), [our Matrix room](https://matrix.to/#/#community:nixos.org), or on the [#nixos IRC channel](https://web.libera.chat/#nixos).
```
Otherwise, people without spaces support will have problems and others will not know which room to join.

Reply:
Actually #community:nixos.org is the space.

Reply:
Oops, should be `#nix:nixos.org`.